tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "moduleResolution": "Node",
+    "declaration": true,
+    "sourceMap": true,
+    "strict": true,
+    "target": "es5",
+    "lib": [
+      "DOM",
+      "ES2016"
+    ],
+    "outDir": "./dist",
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true
```

Comment:
Why is `allowJs` enabled? I noticed below we're also listing `"./src/**/*.js"` as included.

Reply:
I figure this out locally, we're not currently reading these files. See https://github.com/MetaMask/post-message-stream/pull/7#discussion_r493829884 for context.

Reply:
the js has been removed, this is no longer needed :+1: